src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -291,8 +297,147 @@ OCC::Optional<OCC::PinStateEnums::PinState> OCC::CfApiWrapper::PlaceHolderInfo::
     return cfPinStateToPinState(_data->PinState);
 }
 
-OCC::Result<void, QString> OCC::CfApiWrapper::registerSyncRoot(const QString &path, const QString &providerName, const QString &providerVersion)
+QString convertSidToStringSid(void *sid)
+{
+    QString result;
+
+    wchar_t *stringSid = nullptr;
+    if (::ConvertSidToStringSid(sid, &stringSid)) {
+        result = QString::fromWCharArray(stringSid);
+    }
+
+    if (stringSid) {
+        ::LocalFree(stringSid);
+        stringSid = nullptr;
+    }
+
+    return result;
+}
+
+QString retrieveWindowsSID()
+{
+    // FYI: code here is mostly identical to Windows-Classic-Examples/Samples/CloudMirror
+    QScopedPointer<TOKEN_USER> tokenInfo;
+
+    const auto tokenHandle = GetCurrentThreadEffectiveToken();
+
+    DWORD tokenInfoSize = { 0 };
```

Comment:
I'd drop either the = or the curly braces, or move to `auto tokenInfoSize = DWORD{0};` whichever suits you best.